R/pkg/R/mllib.R

```diff
@@ -406,6 +432,8 @@ ml.load <- function(path) {
   jobj <- callJStatic("org.apache.spark.ml.r.RWrappers", "load", path)
   if (isInstanceOf(jobj, "org.apache.spark.ml.r.NaiveBayesWrapper")) {
     return(new("NaiveBayesModel", jobj = jobj))
+  } else if (isInstanceOf(jobj, "org.apache.spark.ml.GeneralizedLinearRegressionWrapper")) {
```

Comment:
For some reason local tests(R/run-tests.sh) is not capturing these failures. Let me fix this and submit the code